base/timing.jl

```diff
@@ -104,13 +104,22 @@ function padded_nonzero_print(value, str)
     end
 end
 
+"""
+    format_bytes(bytes; digits = 3)
+
+Converts from an integer number of bytes to a pretty printed string in nearest significant size units.
 
-function format_bytes(bytes) # also used by InteractiveUtils
+```julia-repl
+julia> format_bytes(123456789)
+"117.738 MiB"
+```
+"""
+function format_bytes(bytes; digits = 3) # also used by InteractiveUtils
```

Comment:
Should probably be `formatbytes` if it's going to be exposed publicly.

Reply:
Makes sense, updated 👍🏻